rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -921,6 +967,7 @@ void RSocketStateMachine::writeNewStream(
 }
 
 void RSocketStateMachine::writeRequestN(StreamId streamId, uint32_t n) {
+  resumeManager_->incrConsumerAllowance(streamId, n);
```

Comment:
I think there is bug here. If the REQUEST_N frame never makes it to the other side, we should not increment the consumer allowance.